vllm/lora/ops/__init__.py

```diff
@@ -0,0 +1,18 @@
+from vllm.triton_utils import HAS_TRITON
+
+if HAS_TRITON:
+    from vllm.lora.ops.triton.bgmv_expand import bgmv_expand
+    from vllm.lora.ops.triton.bgmv_expand_slice import bgmv_expand_slice
+    from vllm.lora.ops.triton.bgmv_shrink import bgmv_shrink
+    from vllm.lora.ops.triton.sgmv_expand import sgmv_expand
+    from vllm.lora.ops.triton.sgmv_expand_slice import sgmv_expand_slice
+    from vllm.lora.ops.triton.sgmv_shrink import sgmv_shrink
+else:
```

Comment:
The Triton ops and Torch ops don't have an if-else relationship， so we should ensure they each have their own separate import

Reply:
Maybe it would be better to organize the ops folders in the following structure?
```shell
├── ops
│   ├── torch_ops
│   └── triton_ops
```

Reply:
Good idea, just changed the names